<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Device</class>
 <widget class="QWidget" name="Device">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>671</width>
    <height>124</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>-1</y>
     <width>441</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>POWER</string>
   </property>
   <widget class="QPushButton" name="rechargeButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>31</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Recharge</string>
    </property>
   </widget>
   <widget class="QLabel" name="chargeStatus">
    <property name="geometry">
     <rect>
      <x>122</x>
      <y>54</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Not charging</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="batteryIndicator">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>30</y>
      <width>91</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="powerButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Power On</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="probeButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>30</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply on skin</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
